File: CRM/src/app/shared/eHospital_DongNai_A/Sys_ErrorLog.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_ErrorLog } from './Sys_ErrorLog.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_ErrorLogService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'ErrorLogID', 'ErrorTime', 'UserName', 'ErrorNumber', 'ErrorSeverity', 'ErrorState', 'ErrorProcedure', 'ErrorLine', 'ErrorMessage'];

    List: Sys_ErrorLog[] | undefined;
    ListFilter: Sys_ErrorLog[] | undefined;
    FormData!: Sys_ErrorLog;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_ErrorLog";
    }
}

File: Service_eHospital_DongNai_A/Implement/Sys_ErrorLogService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class Sys_ErrorLogService : BaseService<Sys_ErrorLog, ISys_ErrorLogRepository>
    , ISys_ErrorLogService
    {
    private readonly ISys_ErrorLogRepository _Sys_ErrorLogRepository;
    public Sys_ErrorLogService(ISys_ErrorLogRepository Sys_ErrorLogRepository) : base(Sys_ErrorLogRepository)
    {
    _Sys_ErrorLogRepository = Sys_ErrorLogRepository;
    }
    }
    }

